src/components/timecard/edit-shift-hours/EditShiftHours.jsx

```diff
@@ -66,43 +96,51 @@ const EditShiftHours = ({
       actualEndTime: actualEndDateTime,
     };
 
-    validateServiceErrors(setServiceError, async () => {
-      const params = new UrlSearchParamBuilder()
-        .setTenantId('00000000-0000-0000-0000-000000000000')
-        .getUrlSearchParams();
-
-      const response = !timeEntry.timeEntryId
-        ? await createTimeEntry(timecardPayload, params)
-        : await updateTimeEntry(timeEntry.timeEntryId, timecardPayload, params);
-
-      if (response?.data?.items?.length > 0) {
-        const responseItem = response.data.items[0];
-        const formattedStartTime = formatTime(responseItem.actualStartTime);
-        const formattedEndTime = responseItem.actualEndTime
-          ? formatTime(responseItem.actualEndTime)
-          : '';
-
-        const newTimeEntries = deepCloneJson(timeEntries);
-        newTimeEntries[timeEntriesIndex] = ContextTimeEntry.createFrom(
-          timeEntry
-        )
-          .setStartTime(formattedStartTime)
-          .setFinishTime(formattedEndTime)
-          .setTimeEntryId(responseItem.id);
-
-        setTimeEntries(newTimeEntries);
-        setSummaryErrors({});
-        setShowEditShiftHours(false);
-      }
-    });
+    validateServiceErrors(
+      setServiceError,
+      async () => {
+        const params = new UrlSearchParamBuilder()
+          .setTenantId('00000000-0000-0000-0000-000000000000')
+          .getUrlSearchParams();
+
+        const response = !timeEntry.timeEntryId
+          ? await createTimeEntry(timecardPayload, params)
+          : await updateTimeEntry(
+              timeEntry.timeEntryId,
+              timecardPayload,
+              params
+            );
+
+        if (response?.data?.items?.length > 0) {
+          const responseItem = response.data.items[0];
+          const formattedStartTime = formatTime(responseItem.actualStartTime);
+          const formattedEndTime = responseItem.actualEndTime
+            ? formatTime(responseItem.actualEndTime)
+            : '';
+
+          const newTimeEntries = deepCloneJson(timeEntries);
+          newTimeEntries[timeEntriesIndex] = ContextTimeEntry.createFrom(
+            timeEntry
+          )
+            .setStartTime(formattedStartTime)
+            .setFinishTime(formattedEndTime)
+            .setTimeEntryId(responseItem.id);
+
+          setTimeEntries(newTimeEntries);
+          setSummaryErrors({});
+          setShowEditShiftHours(false);
+        }
+      },
+      handleServerValidationErrors
```

Comment:
Shouldn't this be "handleServerValidationErrors()"?

Reply:
We are just passing in a reference here so that it can be used inside `validateServiceErrors`